pkg/noobaaaccount/noobaaaccount.go

```diff
@@ -80,12 +81,24 @@ func CmdUpdate() *cobra.Command {
 func CmdRegenerate() *cobra.Command {
 	cmd := &cobra.Command{
 		Use:   "regenerate <noobaa-account-name>",
-		Short: "Regenerate S3 Credentials",
+		Short: "Regenerate S3 Credentials(access-key and secret-key)",
```

Comment:
You added in `CmdRegenerate` instead of `CmdCredentials`.
Anyway, I think you can remove it, because now I understand the whole menu is with "credential" as a known term, so no need to explain.

Reply:
Sure, will update it. Thanks